macros/base/start_print.cfg

```diff
@@ -159,9 +153,7 @@ gcode:
     # If auto z calibration plugin is enabled, we measure the nozzle height using the physical Z endstop probe, followed by
     # a measurement of the probe on the physical Z endstop probe, and then a measurement of the center of the bed.
     {% if purge_and_brush_enabled %}
-        {% if status_leds_enabled %}
-            STATUS_LEDS COLOR="CLEANING"
-        {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.